src/view/pages/FarmerSearch/index.tsx

```diff
@@ -43,13 +45,20 @@ interface DispatchPropsType {
   navigateToFarmer(): void;
 }
 
-type FarmerSearchPropsType = StorePropsType & DispatchPropsType & OwnPropsType;
+/** FarmerSearch OwnStateType */
+interface OwnStateType {}
 
-/** FarmerSearch Props */
-type Props = FarmerSearchPropsType & InjectedSearchProps;
+/** FarmerSearch WrappedPropsType */
+type WrappedPropsType = StorePropsType & DispatchPropsType & OwnPropsType;
+
+/** FarmerSearch PropsType */
+type PropsType = WrappedPropsType & InjectedSearchProps;
+
+/** FarmerSearch StateType */
+type StateType = OwnStateType;
```

Comment:
I would recommend leaving as OwnStateType, StateType could be confused w/ redux state. I agree w/ you that the lack of symmetry looks weird, but it just boils down to the fact that redux has state and not props

Reply:
That makes sense